bvm/ballerina-core/src/main/java/org/ballerinalang/BLangProgramRunner.java

```diff
@@ -92,19 +94,13 @@ private static void initDebugger(ProgramFile programFile, Debugger debugger) {
         }
     }
 
-    public static FunctionInfo getEntryFunctionInfo(PackageInfo entryPkgInfo, String functionName) {
-        String errorMsg = "'" + functionName + "' function not found in '"
-                            + entryPkgInfo.getProgramFile().getProgramFilePath() + "'";
+    public static FunctionInfo getMainFunctionInfo(PackageInfo entryPkgInfo) {
+        String errorMsg = "'main' function not found in '" + entryPkgInfo.getProgramFile().getProgramFilePath() + "'";
```

Comment:
```suggestion
        String errorMsg = "'main' function not found in '" + entryPkgInfo.getProgramFile().getProgramFilePath() + "' module";
```

Reply:
We seem to be printing the path specified by the user here, so saying module may not be correct IMO.

Reply:
Ah, didn't see that it's printing the file path. In that case, we don't need the word `module`.